controls/V-72213.rb

```diff
@@ -1,78 +1,74 @@
+dynamically on accessed files. If this capability is not available, the system
+must be configured to scan, at a minimum, all altered files on the system on a
+daily basis.
 
-    If the system processes inbound SMTP mail, the virus scanner must be configured
-to scan all received mail.
+    If the system processes inbound SMTP mail, the virus scanner must be
+configured to scan all received mail.
   "
   impact 0.7
-  tag "severity": "high"
   tag "gtitle": "SRG-OS-000480-GPOS-00227"
   tag "gid": "V-72213"
-  tag "rid": "SV-86837r1_rule"
+  tag "rid": "SV-86837r2_rule"
   tag "stig_id": "RHEL-07-032000"
-  tag "cci": "CCI-001668"
-  tag "nist": ["SI-3 a", "Rev_3"]
-  tag "check": "Verify the system is using a DoD-approved virus scan program.
+  tag "cci": ["CCI-001668"]
+  tag "documentable": false
+  tag "nist": ["SI-3 a", "Rev_4"]
+  tag "check": "Verify the system is using a virus scan program.
 
 Check for the presence of \"McAfee VirusScan Enterprise for Linux\" with the
 following command:
 
 # systemctl status nails
 nails - service for McAfee VirusScan Enterprise for Linux
 >  Loaded: loaded
-/opt/NAI/package/McAfeeVSEForLinux/McAfeeVSEForLinux-2.0.2.<build_number>; enabled)
+/opt/NAI/package/McAfeeVSEForLinux/McAfeeVSEForLinux-2.0.2.<build_number>;
+enabled)
 >  Active: active (running) since Mon 2015-09-27 04:11:22 UTC;21 min ago
 
-If the \"nails\" service is not active, check for the presence of \"clamav\" on the
-system with the following command:
+If the \"nails\" service is not active, check for the presence of \"clamav\" on
+the system with the following command:
 
 # systemctl status clamav-daemon.socket
  systemctl status clamav-daemon.socket
   clamav-daemon.socket - Socket for Clam AntiVirus userspace daemon
      Loaded: loaded (/lib/systemd/system/clamav-daemon.socket; enabled)
      Active: active (running) since Mon 2015-01-12 09:32:59 UTC; 7min ago
 
-If neither of these applications are loaded and active, ask the System Administrator
-if there is an antivirus package installed and active on the system.
+If neither of these applications are loaded and active, ask the System
+Administrator if there is an antivirus package installed and active on the
+system.
 
 If no antivirus scan program is active on the system, this is a finding."
-  tag "fix": "Install an approved DoD antivirus solution on the system."
+  tag "fix": "Install an antivirus solution on the system."
+  tag "fix_id": "F-78567r2_fix"
 
   describe.one do
-    describe service('nails') do
-      it { should be_running }
+	  describe service('nails') do
+	    it { should be_running }
     end
     describe service('clamav-daemon.socket') do
-      it { should be_running }
-    end
-  end
+	    it { should be_running }
+	  end
+  end if ENABLE_AV
+
+  describe "The system is not required to have AntiVirus Installed" do
+    skip "The system does not require AntiVirus to be enabled"
+  end if !ENABLE_AV
```

Comment:
This check seems odd. Shouldn't the user have an override in their inheriting profile if they want to disable different checks?

If not, then this pattern should literally be everywhere (which I think is a bad idea). Suggest removing the disable capability.

Reply:
@trevor-vaughan @aaronlippold That's a really good point. I think that a similar pattern makes sense in cases where you want to be able to disable part of a control or if you determine dynamically that a control is Not Applicable. In cases like this, I agree that it should probably be removed.

Reply:
Hi, for me it was a matter of level of complexity. There were only one or two cases that we kept running into with various sponsors. Generally speaking the disabled slow controls and disabled av was to more easily support some of the cloud platforms and government environments. Obscenely I agree with your concern and formally best way would be to do everything in an overlay if we were to modify the base profile but that would add a large sprawl of overlays which we would then have to maintain. I think we should try to find a middle ground so that the base itself supported at least supported some of these cases but we agreed that we didn't go to far in the feature creep with this.